gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/client/graph/ValidateGraphHostOperation.java

```diff
@@ -49,19 +52,23 @@ public void execute() throws GraphOperationException {
             validateSystemIsUp(response);
 
         } catch (final WebClientRequestException e) {
+            LOGGER.error("Get Status request for '" + proxySubGraph.getGraphId() + "' failed. Reason: " + e.getMostSpecificCause().getMessage() + " at " + e.getUri(), e);
```

Comment:
Again, parameteriszed logging would make this a lot more readable and more effecient.

Reply:
Changed as per the review comments